src/ast/postprocessing/mapping.ts

```diff
@@ -12,21 +12,25 @@ import { UserDefinedTypeName } from "../implementation/type/user_defined_type_na
 import { BuiltinReferencedDeclarationNormalizer } from "./builtin_referenced_declaration_normalizer";
 import { StructuredDocumentationReconstructingPostprocessor } from "./structured_documentation_reconstruction";
 
-const reconstructor = new StructuredDocumentationReconstructingPostprocessor();
-const refNormalizer = new BuiltinReferencedDeclarationNormalizer();
+const reconstructor = new StructuredDocumentationReconstructingPostprocessor(1000);
+const refNormalizer = new BuiltinReferencedDeclarationNormalizer(100);
+
+function sortByPriority(a: ASTNodePostprocessor, b: ASTNodePostprocessor): number {
+    return a.priority - b.priority;
+}
 
 export const DefaultPostprocessorMapping = new Map<
     ASTNodeConstructor<ASTNode>,
     ASTNodePostprocessor[]
 >([
-    [ContractDefinition, [reconstructor]],
-    [EventDefinition, [reconstructor]],
-    [FunctionDefinition, [reconstructor]],
-    [ModifierDefinition, [reconstructor]],
-    [VariableDeclaration, [reconstructor]],
+    [ContractDefinition, [reconstructor].sort(sortByPriority)],
```

Comment:
Yes... This is mostly to not forget that this aray have to be sorted-by-priority initially. Otherwise we will have to sort it on runtime and that would consume more time.

Reply:
Addressed in https://github.com/ConsenSys/solc-typed-ast/pull/56/commits/77d1a8134ac36f4df3ef052e8412481117dfc4d4